### DISCUSSION TITLE
How to shut down a dbeaver instance.

### DISCUSSION THREAD

**Discussion by cmpds** [dbeaver/dbeaver#20870] [Q&A]
Hello,  I am a fairly new and infrequent user of dbeaver, please forgive me if I don't use the correct terminology.  Yesterday, I was attempting to export a csv file and the export wizard locked up when I attempted to change some of the export settings.  All options in the wizard and the entire window including the menu bar became grayed out.  I couldn't cancel the export or exit my session.  After waiting awhile, I just X'd out of the session hoping it would terminate the session.  Now when I try to open dbeaver, I get a message that there is an instance running.  I followed the troubleshooting instructions to manually reset workspace, but I still get the same warning.  I attempted to uninstall dbeaver, but it didn't work because there was an instance running. I also tried to delete the folder, but that didn't work either.  I installed the latest version of dbeaver in a new folder to try and by-pass the problem, but when I tried to open that version from the new folder, I still get the same warning.  How do you shut down an instance that you can't access?

![image](https://github.com/dbeaver/dbeaver/assets/141945064/448989af-9375-476a-aca5-f74f9e558d45)

Thank you for any help with this.

**Comment by E1izabeth** [comment]
We haven't experienced such issue. Have you tried to restart your PC?

**Reply by cmpds** [reply]
I am working on a corporate Virtual Machine that I can't shut down.  They reset it every weekend, so I'll see what happens on Monday.

**Comment by E1izabeth** [comment]
Have you tried to press `Ignore` button?

**Reply by cmpds** [reply]
Yes, it opens another instance, but the other one is always still running.
